test/unit-tests/expression/function/help.test.js

```diff
@@ -2,6 +2,52 @@ import assert from 'assert'
 import math from '../../../../src/defaultInstance.js'
 import { embeddedDocs } from '../../../../src/expression/embeddedDocs/embeddedDocs.js'
 
+let mathDocs = math.create(math.all)
+const originalConfig = mathDocs.config()
+// Add functions to the skipDocs array if their examples in the embedded docs contain acceptable errors
```

Comment:
should this comment also say 'or if they are not intended to have embedded docs at all'?